.dockerignore

```diff
@@ -0,0 +1,9 @@
+# Files to ignore when building docker image
+*.log
+logs/
+./benchmark
+.DS_Store
+*.egg-info
+*logging
+./docker/Dockerfile*
+Anaconda3-2020.11-Linux-x86_64.sh
```

Comment:
Can we remove this `Anaconda3-2020.11-xxx`. it looks too hard coded. At least, we should use `*` instead of detailed version.

Reply:
This is the Anaconda install script that will be downloaded by `install.sh` if the host machine has no conda environment. But yes it is better to include wildcard `*`.